src/frontend/components/UI/Dialog/MessageBoxModal/index.tsx

```diff
@@ -0,0 +1,50 @@
+import './index.css'
+import React from 'react'
+import {
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogHeader
+} from 'frontend/components/UI/Dialog'
+
+interface MessageBoxModalProps {
+  title: string
+  message: string
+  onClose: () => void
+  buttons: Array<string>
+  buttonsOnClick: Array<() => void>
```

Comment:
Combine this to a map or new interface ?
`Map<string, Function>` or 
```typescript
interface Button {
  name: string
  action: () => void
}
```

Reply:
this was a great suggestion. simplified the code a bit and will allow us to add other button options in the future like a styling enum by only changing the code in one place